libd-llvm/src/d/llvm/type.d

```diff
@@ -103,7 +104,17 @@ final class TypeGen {
 				return LLVMPointerType(LLVMInt8TypeInContext(llvmCtx), 0);
 		}
 	}
-	
+
+	LLVMTypeRef getSizeT() {
+		static LLVMTypeRef sizeT;
```

Comment:
Yes it also introduce global state preferably in an hidden manner, and it going to be even slower.